hazelcast-sql/src/main/codegen/templates/Parser.jj

```diff
@@ -0,0 +1,8410 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
```

Comment:
I'm not sure we can do this. We can't re-license the file when we've taken it in full. I'd ask someone to confirm.